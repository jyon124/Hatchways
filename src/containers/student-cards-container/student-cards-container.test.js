describe("Student Cards Container Component Tests", () => {
    it.todo("Renders a Student Cards Container component");
    it.todo("Renders Student Card components when students data are available");
    it.todo(
        "Test if each student data properly init.todoialized wit.todoh empty tag array"
    );
    it.todo("Renders StudentCards that only match the name search term");
    it.todo("Renders StudentCards that only match the tag search term");
    it.todo(
        "Renders StudentCards that only match the search & tag search term"
    );
    it.todo(
        "Check if StudentCards are sorted by most relevant to the name search input"
    );
    it.todo(
        "Check if StudentCards are sorted alphabetically in case the same name search term exists in different cards"
    );
});
