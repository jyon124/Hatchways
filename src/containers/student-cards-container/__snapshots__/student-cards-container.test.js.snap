// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Cards Container Component Tests Renders a Student Cards Container component 1`] = `"<div class=\\"studetCardContainerWrapper\\"><div class=\\"contentContainer\\"><input class=\\"input\\" placeholder=\\"Search by name\\"><input class=\\"input\\" placeholder=\\"Search by tag\\"></div></div>"`;
